templates/command/new/headersOnly/bitbucket-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+clone:
+  depth: full
+
+definitions:
+  steps:
+    - step: &Versioning
+        name: Versioning
+        script:
+          - pipe: docker://hemelectronics/pipe-gitflow-versioning:0.1.3
+            variables:
+              BITBUCKET_USERNAME: "${BITBUCKET_USERNAME}"
+              BITBUCKET_PASSWORD: "${BITBUCKET_PASSWORD}"
+        artifacts:
+          - .version/*
+        services:
+          - docker
+        caches:
+          - docker
+    - step: &CreatePackage
+        name: Build
+        image:
+          name: hemelectronics/buildcpp-native-gcc:3.0.0
+          run-as-user: 1000
+        script:
+          - echo "Build running"
+          - CHANNEL=$(python3 ./getChannel.py)
+          - conan create . hem/"$CHANNEL"
```

Comment:
czy "" nie przeszkadzają przypadkiem?

Reply:
to polecenie bashowe, więc no chyba są zbędne